travis/getaddons.py

```diff
@@ -10,12 +10,14 @@
 import sys
 
 
+manifest_files = ['__openerp__.py', '__odoo__.py', '__terp__.py']
+
+
 def is_module(path):
     if not os.path.isdir(path):
         return False
-    manifs = ['__openerp__.py', '__odoo__.py', '__terp__.py', '__init__.py']
     files = os.listdir(path)
-    filtered = [x for x in files if x in manifs]
+    filtered = [x for x in files if x in (manifest_files + ['__init__.py'])]
```

Comment:
I separated the list of possible manifest files from the module file in order to recycle this list in `get_test_dependencies`, but from the point of view of `is_module`, yes, no change of behavior (if I didn't screw up)